support/Pipelines/Bbh/Ringdown.py

```diff
@@ -242,27 +249,37 @@ def start_ringdown(
     ringdown_params["ShapeMapLMax"] = int(ringdown_ylm_coefs[0][4])
     ringdown_params["PathToAhCCoefsH5File"] = path_to_output_h5
     ringdown_params["AhCCoefsSubfilePrefix"] = output_subfile_prefix
-    ringdown_params["Rotation0"] = rotation_func_with_2_derivs[0][0]
-    ringdown_params["Rotation1"] = rotation_func_with_2_derivs[0][1]
-    ringdown_params["Rotation2"] = rotation_func_with_2_derivs[0][2]
-    ringdown_params["Rotation3"] = rotation_func_with_2_derivs[0][3]
-    ringdown_params["dtRotation0"] = rotation_func_with_2_derivs[1][0]
-    ringdown_params["dtRotation1"] = rotation_func_with_2_derivs[1][1]
-    ringdown_params["dtRotation2"] = rotation_func_with_2_derivs[1][2]
-    ringdown_params["dtRotation3"] = rotation_func_with_2_derivs[1][3]
-    ringdown_params["dt2Rotation0"] = rotation_func_with_2_derivs[2][0]
-    ringdown_params["dt2Rotation1"] = rotation_func_with_2_derivs[2][1]
-    ringdown_params["dt2Rotation2"] = rotation_func_with_2_derivs[2][2]
-    ringdown_params["dt2Rotation3"] = rotation_func_with_2_derivs[2][3]
-    ringdown_params["ExpansionOuterBdry"] = (
-        expansion_func_outer_boundary_with_2_derivs[0]
-    )
-    ringdown_params["dtExpansionOuterBdry"] = (
-        expansion_func_outer_boundary_with_2_derivs[1]
-    )
-    ringdown_params["dt2ExpansionOuterBdry"] = (
-        expansion_func_outer_boundary_with_2_derivs[2]
-    )
+    ringdown_params["Rotation0"] = evaluated_fot_dict["Rotation"][0][0]
+    ringdown_params["Rotation1"] = evaluated_fot_dict["Rotation"][0][1]
+    ringdown_params["Rotation2"] = evaluated_fot_dict["Rotation"][0][2]
+    ringdown_params["Rotation3"] = evaluated_fot_dict["Rotation"][0][3]
```

Comment:
Let's try to figure out how to write lists of lists to yaml so we can avoid this. If it turns out that we can't, then we can just do this except using for loops like
```py
for i in range(4):
  for j in range(3):
    prefix = "" if j == 0 else ("dt" if j == 1 else "dt2")
    ringdown_params[f"{prefix}Rotation{i}"] = evaluated_fot_dict["Rotation"][j][i]
    if i == 4:
      continue
    # same for expansion and translation